ci/sync-protobuf.sh

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env bash
+
+set -eou pipefail
+
+# syn-protobuf.sh is a bash script to sync the protobuf
```

Comment:
Should we keep this in the ci/ folder since we're not using it?

Reply:
Let's move it to the `scripts/` folder?